README.md

```diff
@@ -37,6 +37,14 @@ This installs:
 * The `buf`, [`protoc-gen-buf-breaking`][breaking], and [`protoc-gen-buf-lint`][lint] binaries
 * Shell completion scripts for [Bash], [Fish], [Powershell], and [zsh]
 
+### Scoop
```

Comment:
I don't know if this is something we add to the main README.md, we already reference below to see the docs for installing on windows

Reply:
Fair enough